### DISCUSSION TITLE
Unable to login with custom JWT provider (iOS SDK)

### DISCUSSION THREAD

**Discussion by nickblack1919** [Web3Auth/Web3Auth#1354] [Help]
Trying to perform custom login with JWT obtained from AWS Cognito and always get the following error:
"Unable to detect login share from the Auth Network..."

Tried all kinds of combinations setting redirect_uri, verifierIdField, client_id in the ExtraLoginOptions. 
Same verifier works from the Web App example.
Please help.

Code: 

```
Static.shared.web3Auth = await Web3Auth(
					W3AInitParams(
						clientId: clientId,
						network: network,
						loginConfig: [
							Web3AuthProvider.JWT.rawValue: .init(
								verifier: "savechain-testnet-cognito2",
								typeOfLogin: TypeOfLogin.jwt,
								name: "savechain-testnet-cognito2",
								clientId: clientId
							)
						]))
let result = try await Static.shared.web3Auth?.login(
	W3ALoginParams(loginProvider: .JWT,
                                       extraLoginOptions: ExtraLoginOptions(display: nil, prompt: nil, max_age: nil, ui_locales: nil,
id_token_hint: nil, id_token: ID_TOKEN, login_hint: nil, acr_values: nil, scope: nil, audience: nil, connection: nil, domain: nil, 
client_id: nil, redirect_uri: nil, leeway: nil, verifierIdField: "username", isVerifierIdCaseSensitive: nil))
				)
```
<img width="344" alt="Web3Auth_jwt_error" src="https://user-images.githubusercontent.com/20976653/220199462-70855ef3-21d8-4008-b669-4d325a6730c7.png">

**Comment by YZhenY** [comment]
Could you share the console error youre getting?

**Comment by ihsraham** [comment]
@nickblack1919 Hey, are you still getting this error?
